libres/lib/enkf/enkf_obs.cpp

```diff
@@ -317,18 +316,15 @@ static void enkf_obs_get_obs_and_measure_summary(
         if (step < 0)
             break;
 
-        if (obs_vector_iget_active(obs_vector, step)) {
```

Comment:
We are guaranteed that this is active because we loop through active steps?

Reply:
> We are guaranteed that this is active because we loop through active steps?

Yes. This was a bit "belte and bukseseler" - using the same underlying mechanism.